tests/lax_numpy_test.py

```diff
@@ -3696,7 +3696,7 @@ def testIssue764(self):
   def testIssue776(self):
     """Tests that the scatter-add transpose rule instantiates symbolic zeros."""
     def f(u):
-      y = jax.ops.index_add(np.ones(10,), [2, 4, 5], u)
+      y = jax.ops.index_add(np.ones(10,), np.array([2, 4, 5]), u)
```

Comment:
I'd actually suggest this should be `jax.ops.index[2, 4, 5]`.

It's never been intentional that you can pass an array here...

Reply:
I think that's something different, right? In numpy it would be akin to
```python
x[2, 4, 5] += u
```
which results in an index error for 1D arrays.

The intent here is broadcasting indexing along one dimension; something like:
```python
idx = np.array([2, 4, 5])
x[idx] += u
```

Reply:
Then it probably should be `jax.ops.index[np.array([2, 4, 5])]`.

Although perhaps
`x.at[np.array([2, 4, 5])].set(y)` is even better?

(We should probably prefer that syntax everywhere.)

Reply:
Done.